app/views/shared/_nav_branded.html.slim

```diff
@@ -2,7 +2,8 @@ nav.py2.sm-pt3.sm-pb0.bg-light-blue.center
   .my1.sm-mb0.relative
     = image_tag(asset_url('logo.svg'), width: 124,
       alt: APP_NAME, class: 'align-middle')
-    .px2.inline-block
-      span.absolute.top-0.bottom-0.border-right
-    = image_tag(asset_url("sp-logos/#{@sp_logo}"), height: 50,
-      alt: @sp_name, class: 'align-middle')
+    - if @sp_logo
```

Comment:
I think we could use the `decorated_session` here, so this would be something like:

`= render decorated_session.service_provider_logo_partial`

and then we'd have the content in a separate partial and use the decorators like this:

```
# app/decorators/service_provider_session_decorator.rb

def service_provider_logo_partial
  'shared/sp_logo'
end
```

```
# app/decorators/session_decorator.rb

def service_provider_logo_partial
  'shared/null'
end
```